### ISSUE TITLE
ocropus-visualize-results

### ISSUE THREAD

**Issue by amitdo** [tmbdev/ocropy#202]
It should be updated or moved to the OLD folder.

**Comment by zuphilip** [comment]
Is there a problem with `ocropus-visualizu-results`?

It is part of the automatic test and we are passing currently: [![Build Status](https://travis-ci.org/tmbdev/ocropy.svg)](https://travis-ci.org/tmbdev/ocropy).

**Comment by amitdo** [comment]
I just looked at the code and the lines with the 'csegs' made me think it's part of the 0.6 code.
Stupid mistake. Sorry about that. closing.
.

**Comment by zuphilip** [comment]
Actually, I think this part of the code should be update. There are no `csegs` files created nowadays and there is a dependency on `linerec` which was moved to the "OLD" folder a long time ago: https://github.com/tmbdev/ocropy/blob/97ace5581b4c4deb95cfcd17b595b0ff1da36c25/OLD/linerec.py . Moreover, the `rseg` seems also to be outdated. @amitdo Should we delete the lines related to the not anymore used file formats `csegs` and `rseg`?

**Comment by amitdo** [comment]
I wonder what Tom meant here:
https://github.com/tmbdev/ocropy/issues/126#issuecomment-257213053

**Comment by amitdo** [comment]
Do you know how this kind of image can be produced?
http://www.danvk.org/images/ocropus/new-network.png

**Comment by zuphilip** [comment]
Indeed, I don't know where these two formats are still used. We can also leave the code as it is there. But testing for these old formats will not be possible then.

The picture of @danvk looks for me like an adaption of the output of `ocropus-rpred -S ...` which looks for example like this:

![010030 rpred debug](https://cloud.githubusercontent.com/assets/5199995/25519418/0829a972-2bf8-11e7-9eec-6bec64daeb47.png)
